src/MagnumPlugins/UfbxImporter/UfbxImporter.cpp

```diff
@@ -1873,38 +1890,12 @@ Containers::Optional<AnimationData> UfbxImporter::doAnimation(UnsignedInt id) {
         #endif
         default: CORRADE_INTERNAL_ASSERT_UNREACHABLE(); /* LCOV_EXCL_LINE */
         }
-
-        std::size_t valuesSize = valueBuffer.size() - valueOffset;
-        arrayAppend(animTracks, {
-            prop.nodeId - _state->nodeIdOffset,
-            target,
-            trackType,
-            keyCount,
-            timeOffset,
-            valueOffset,
-            valuesSize,
-            valueElementSize,
-        });
     }
 
-    Containers::Array<AnimationTrackData> tracks{DefaultInit, animTracks.size()};
-
-    Containers::ArrayView<Float> timeData;
-    Containers::ArrayView<char> valueData;
-    Containers::Array<char> data = Containers::ArrayTuple{
-        {NoInit, timeBuffer.size(), timeData},
-        {NoInit, valueBuffer.size(), valueData},
-    };
-
-    Utility::copy(timeBuffer, timeData);
-    Utility::copy(valueBuffer, valueData);
-
     for(std::size_t i = 0; i < animTracks.size(); ++i) {
         const AnimTrack& animTrack = animTracks[i];
 
-        Containers::ArrayView<Float> times = timeData.sliceSize(animTrack.timeOffset, animTrack.keyCount);
-        Containers::ArrayView<char> valueBytes = valueData.sliceSize(animTrack.valueOffset, animTrack.valuesSize);
-        Containers::StridedArrayView1D<const void> stridedValues{valueBytes, times.size(), animTrack.valueElementSize};
+        Containers::StridedArrayView1D<const void> values{animTrack.values.asContiguous(), animTrack.values.size()[0], animTrack.values.stride()[0]};
```

Comment:
There's probably a nicer way to do this but this is the best I could come up with

Reply:
Yes, 2D, updated the message, sorry. 

I'm answering 15 threads at once and can't keep my thoughts straight :see_no_evil:

Reply:
I changed my mind after seeing how many constructors there already is -- don't really want to basically double their count again :sweat_smile: So `transposed<0, 1>()[0]` it is.

Reply:
Seems to work flawlessly, nice!